go/vt/wrangler/reparent.go

```diff
@@ -917,21 +917,22 @@ func (wr *Wrangler) emergencyReparentShardLocked(ctx context.Context, ev *events
 	event.DispatchUpdate(ev, "stop replication on all replicas")
 	wg := sync.WaitGroup{}
 	mu := sync.Mutex{}
-	statusMap := make(map[string]*replicationdatapb.Status)
+	statusMap := make(map[string]*replicationdatapb.StopReplicationStatus)
 	for alias, tabletInfo := range tabletMap {
 		wg.Add(1)
 		go func(alias string, tabletInfo *topo.TabletInfo) {
 			defer wg.Done()
 			wr.logger.Infof("getting replication position from %v", alias)
 			ctx, cancel := context.WithTimeout(ctx, waitReplicasTimeout)
 			defer cancel()
-			rp, err := wr.tmc.StopReplicationAndGetStatus(ctx, tabletInfo.Tablet)
+			// TODO: Once we refactor EmergencyReparent, change the stopReplicationOption argument to IOThreadOnly.
+			_, stopSlaveStatus, err := wr.tmc.StopReplicationAndGetStatus(ctx, tabletInfo.Tablet, replicationdatapb.StopReplicationMode_IOANDSQLTHREAD)
```

Comment:
```suggestion
			_, status, err := wr.tmc.StopReplicationAndGetStatus(ctx, tabletInfo.Tablet, replicationdatapb.StopReplicationMode_IOANDSQLTHREAD)
```

Reply:
Adjusted, although I picked `stopReplicationStatus` because I think it's more descriptive.